orangecontrib/timeseries/widgets/owmovingtransform.py

```diff
@@ -205,6 +206,11 @@ class Warning(widget.OWWidget.Warning):
         window_to_large = widget.Msg("Window width is to large")
```

Comment:
I didn't notice this before, but it should be "Window width is too large."
Same below.

Reply:
👍